test/fake_http_server.cc

```diff
@@ -69,6 +69,15 @@ void FakeServer::AllowStream(const std::string& path) {
   handler_.path_streams[path];
 }
 
+int FakeServer::NumWatchers(const std::string& path) {
+  auto stream_it = handler_.path_streams.find(path);
+  if (stream_it == handler_.path_streams.end()) {
+    LOG(ERROR) << "Attempted to wait for an unknown path " << path;
+    return false;
```

Comment:
Why are we returning false here, instead of 0?

Reply:
Because it was a copy/paste of the code in `WaitForMinTotalConnection`, and C++ freely coerces between `bool` and `int`, so there was no compiler warning. The error doesn't make much sense either, so I fixed both.